mojo/core/node_controller.cc

```diff
@@ -368,19 +368,17 @@ void NodeController::SendBrokerClientInvitationOnIOThread(
     // Unix Domain Socket
     PlatformChannel node_channel;
     node_connection_params = ConnectionParams(node_channel.TakeLocalEndpoint());
-    // BrokerHost owns itself.
-    BrokerHost* broker_host =
```

Comment:
BrokerHost can communicate with Broker using by UNIX domain socket.
How about using BrokerHost instead of BrokerCastanets for IPC?

Reply:
In that case, when --enable-forking is enabled, BrokerHost will be running on the browser side while BrokerCastanets will be running on the renderer side. We expect that will be working well. I'll modify and apply that.

Reply:
Done. One diff has been added in BrokerHost due to the BUFFER_INIT message. If any other diff occurs in BrokerHost further, we need to consider using BrokerCastanets on both processes regardless of the connection type.